tasks/last_boot_time_nix.json

```diff
@@ -1,3 +1,8 @@
 {
-  "description": "Gets the last boot time of a Linux system"
+  "description": "Gets the last boot time of a Linux system",
+  "private": true,
```

Comment:
Why are these tasks private?

Reply:
I think it’s in the commit message. They’re specific implementations of the task, so we hide them for Bolt.

Reply:
Sorry glanced over description in commit message. I brought it up because I thought I remember discussing if we should make platform specific implementations private for other modules bundled with bolt. For example the `service::{windows,linux}`. I thought we talked about only hiding tasks if they are "sensitive" in some way.